api/buildings/admin.py

```diff
@@ -4,6 +4,20 @@
 from . import models
 
 
+@admin.register(models.Statistic)
+class StatisticAdmin(admin.ModelAdmin):
+    def has_add_permission(self, request):
+        base_add_permission = super(StatisticAdmin, self).has_add_permission(
+            request
+        )
+        if base_add_permission:
+            # if there's already an entry, do not allow adding
+            count = models.Statistic.objects.count()
+            if count == 0:
```

Comment:
It would be great to keep the number of comments to a minimum
One suggestion is to change from checking the value of an `int`
```
count = models.Statistic.objects.count()
if count == 0:
``` 
to checking a `boolean` 
```
has_statistic_entry = models.Statistic.objects.count() != 0
if not has_statistic_entry:
```
A better name probably exists for that variable; still, I think it's better than having a comment